src/meshes/t8code_mesh.jl

```diff
@@ -586,13 +589,525 @@ function adapt!(mesh::T8codeMesh, adapt_callback; recursive = true, balance = tr
     return nothing
 end
 
-# TODO: Just a placeholder. Will be implemented later when MPI is supported.
-function balance!(mesh::T8codeMesh, init_fn = C_NULL)
+"""
+    balance!(mesh::T8codeMesh)
```

Comment:
```suggestion
    Trixi.balance!(mesh::T8codeMesh)
```
We don't `export` it, do we?